COMPATIBILITY.md

```diff
@@ -16,3 +17,11 @@ The below matrix shows the compatibility of the `opensearch-net` with versions o
 | 1.2.4 | 1.0.0 |
 | 1.3.0 | 1.0.0 |
 | 1.3.1 | 1.0.0 |
+| 1.3.2 | 1.0.0 |
+| 1.3.3 | 1.0.0 |
+| 2.0.0 | 1.0.0 |
+| 2.0.1 | 1.0.0 |
+
+## Upgrading
+
+Major versions of OpenSearch introduce breaking changes that require careful upgrades of the client. While `opensearch-net` client 2.0.0 works against the latest OpenSearch 1.x, certain deprecated features removed in OpenSearch 2.0 have also been removed from the client. Please refer to the [OpenSearch documentation](https://opensearch.org/docs/latest/clients/index/) for more information.
```

Comment:
This is a non-breaking change of client, so I think version 2.0 should be 1.1.

Reply:
New version number (1.1 or 2.0) is being discussed, I will update the matrix if needed.

Reply:
I think the discussion was whether it is worth the effort to make next client version compatible with OpenSearch 1.x and 2.x. If so, then name it 1.1.0, otherwise it would be 2.0.0. This PR seems is already maintaining compatibility, so next client version should be 1.1.0?

Reply:
Oh I think you have to, see https://github.com/opensearch-project/opensearch-clients/issues/17. Otherwise users can't upgrade without downtime..

Reply:
Thanks, we're planning to test this somehow. As you can see, not APIs were removed or blocked in this PR.